styles/rating.css

```diff
@@ -0,0 +1,11 @@
+.rating {
+  display: inline-flex;
+  gap: 6px;
+  padding: 0.5rem 1rem;
```

Comment:
Suggestion: you could use px units instead of rem when your value is less than 10px.

Reply:
done